js/index.js

```diff
@@ -79,3 +79,36 @@ const editButton = document.createElement('button');
     messageList.appendChild(newMessage);
     messageForm.reset();
 })
+
+// Projects section by AJAX and JSON
+let githubRequest = new XMLHttpRequest(); 
+githubRequest.open('GET', "https://api.github.com/users/ofeterovskaya/repos");
+githubRequest.send();
+
+githubRequest.addEventListener("load",(e) =>{
+    let repositories = JSON.parse(githubRequest.response);
+    console.log(repositories);
```

Comment:
Nitpick: You'll want to remove this console.log statement to avoid printing out the response to the console for every page load.